packages/client/hmi-client/src/types/workflow.ts

```diff
@@ -6,6 +6,12 @@ export enum WorkflowStatus {
 	ERROR = 'error'
 }
 
+export enum VIEWOPTIONS {
```

Comment:
If we like this approach of viewoptions please let me know if youd like me to rename the options below 
The idea being the view for when its just the node, the view when its clicked on (preview) and the view it is when its the pull page